db/migrate/20190408004639_add_avail_sun_am_to_volunteers.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailSunAmToVolunteers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :volunteers, :avail_sun_am, :string
```

Comment:
Since the user is either available or not, it makes more sense to use a boolean than a string.  All of the changes to add columns could be done in one migration file.